src/Microsoft.AspNetCore.Mvc.TagHelpers/Cache/DistributedCacheTagHelperFormattingContext.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.Mvc.TagHelpers.Cache
+{
+    /// <summary>
+    /// Represents an object containing the information to serialize with <see cref="IDistributedCacheTagHelperFormatter" />.
+    /// </summary>
+    public class DistributedCacheTagHelperFormattingContext
+    {
+        /// <summary>
+        /// Gets the <see cref="TagHelperContext"/> instance.
+        /// </summary>
+        public TagHelperContext Context {get; set; }
+        
+        /// <summary>
+        /// Gets the <see cref="HtmlString"/> instance.
+        /// </summary>
+        public HtmlString HtmlString { get; set; }  
```

Comment:
`HtmlEncodedString`? I had a similar comment in one of my Razor PRs, sounded like we were moving away from `HtmlString`. Could also just do `IHtmlContent`